.github/workflows/preview-env-teardown.yml

```
@@ -0,0 +1,56 @@
+name: preview-env-teardown
+on:
+  pull_request:
+    types: [unlabeled, closed]
+
+jobs:
+  tear-dowm-preview-env:
```

```suggestion
  tear-down-preview-env:
```

non-blocking, but we'll end up correcting this spelling at some point anyway. If it's not inconvenient we might as well fix it right away.